### ISSUE TITLE
[Question] How I can save Dictionary to the iCloud ?

### ISSUE THREAD

**Issue by YHSX88** [phamtanlong/unity-ios-keychain-plugin#2]
Hi 

As per the Apple docs, it is also possible to save dictionary to iCloud Keychain.
So can you please let me know what / how I can do it with your plugin ?

**Comment by rico345100** [comment]
Yes, the documentation is too lacks information. It said it could be used to store game data but doesn't let us know how to do that.
